ansible/roles/buildmaster/templates/master.cfg.j2

```diff
@@ -135,12 +140,20 @@ bash -c 'for repo in {hostdir}/binpkgs {hostdir}/binpkgs/debug {hostdir}/binpkgs
 		descriptionDone=["Finished removing obsolete packages"],
 		workdir='.', haltOnFailure=True, usePTY=True, timeout=14400,
 		decodeRC={0:SUCCESS,1:FAILURE,2:FAILURE})
+	remove_obsoletes_multilib_step = ShellCommand(command=RemoveObsoletesMultilib, logEnviron=False,
+		description=["Removing obsolete multilib packages"],
+		descriptionDone=["Finished removing obsolete multilib packages"],
+		workdir='.', haltOnFailure=True, usePTY=True, timeout=14400,
+		decodeRC={0:SUCCESS,1:FAILURE,2:FAILURE})
 
 	m['bulk_factory_steps'] = [
 		bulk_clean_step, git_clean_step, bootinst_step,
 		bootup_step, conf_step, get_pkgs_step, make_step,
 		remove_obsoletes_step
 	]
+	if m['mach'] == "i686":
+		m['bulk_factory_steps'].append(remove_obsoletes_multilib_step)
```

Comment:
Maybe add a comment explaining why it's necessary?

"Since i686 writes to this repo, it should be the one to clean it up, otherwise it can race"

Reply:
added